modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -299,9 +307,9 @@ export default {
         container: this.state.container.focus ? this.classes.focus
           : (this.state.container.highlight ? this.classes.highlight : null),
```

Comment:
this line also needs to know whether a nonForeign is hovered so we can make the dashed lines appear
`: ((this.state.container.highlight || this.nonForeignHovered) ? this.classes.highlight : null),`